core/server_config.go

```diff
@@ -279,12 +282,30 @@ func FlagSet() *pflag.FlagSet {
 	flagSet.String("network.truststorefile", "", "Deprecated: use 'tls.truststorefile'. PEM file containing the trusted CA certificates for authenticating remote gRPC servers.")
 	flagSet.Int("network.maxcrlvaliditydays", 0, "Deprecated: use 'tls.crl.maxvaliditydays'. The number of days a CRL can be outdated, after that it will hard-fail.")
 
+	// Flags for blacklist features
+	flagSet.Int("pki.blacklist.maxupdatefailhours", 4, "maximum number of hours that a blacklist update can fail")
```

Comment:
For intervals we often use a `time.Duration`, which allows you to write `4h`

Reply:
I'll keep this in mind for future use cases, as it does sound handy